src/main/scala/com/microsoft/hyperspace/index/IndexLogEntry.scala

```diff
@@ -151,7 +170,7 @@ object Directory {
     val leafDirToChildrenFiles = files.toArray.groupBy(_.getPath.getParent)
 
     // Hashmap from directory path to Directory object, used below for quick access from path.
-    val pathToDirectory = HashMap[Path, Directory]()
+    val pathToDirectory = mutable.HashMap[Path, Directory]()
```

Comment:
could you please revert this (and also the import statement) unless specific reason?

Reply:
It was because intellij suggested this. I'll revert this in new PRs